src/elementary_operators.jl

```diff
@@ -214,7 +225,7 @@ function laplacian_operator(
 end
 
 function identity_operator(s::IndsNetworkMap; kwargs...)
-  return stencil(s, [0.0, 1.0, 0.0], 0; kwargs...)
+  return stencil(s, [0.0, 0.0, 1.0, 0.0, 0.0], 0; kwargs...)
```

Comment:
Good spot. I wonder if we should simplify further and avoid using the stencil here.

We could do:

```
  operator_inds = union_all_inds(siteinds(s), prime(siteinds(s)))
  return ITensorNetwork(Op("I"), operator_inds)
```

which should return the identity and be much more efficient.